x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/advanced_editor.tsx

```diff
@@ -159,6 +164,25 @@ export const RangePopover = ({
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiFormRow>
+      <EuiFormRow>
+        <LabelInput
```

Comment:
Hey @flash1293, I saw this one got merged already, but you make this input `compressed` since it lives in a popover and will then match the range inputs better.